sdk/storage/azure-storage-file-share/tests/test_handle_async.py

```diff
@@ -61,10 +61,11 @@ def _validate_handles(self, handles):
         self.assertIsNotNone(handles[0].client_ip)
         self.assertIsNotNone(handles[0].open_time)
 
+    @pytest.mark.skip(reason="Needs further investigation.")
```

Comment:
I don't see this on your list for further investigation :). If you don't plan to investigate it now, we should at least open an Issue to track all the "needs further investigation" skips.